lxcfs.c

```diff
@@ -884,13 +892,16 @@ int main(int argc, char *argv[])
 	}
 
 	newargv[cnt++] = argv[0];
-        if (debug) {
-                newargv[cnt++] = "-d";
-        } else {
-                newargv[cnt++] = "-f";
-        }
+	if (debug) {
+		newargv[cnt++] = "-d";
+	} else {
+		newargv[cnt++] = "-f";
+	}
```

Comment:
I know they have been there before but since you're making that change can you please remove the brackets around the single line if-statements. So:
```
	if (debug)
		newargv[cnt++] = "-d";
	else
		newargv[cnt++] = "-f";
```